tools/localization_analysis/tools/run_map_matcher.cc

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2017, United States Government, as represented by the
```

Comment:
I think this was recently merged to dev right? Do you know if you made changes to that version?

Reply:
I just added more flags for run_map_matcher